### ISSUE TITLE
[Feature] Where is the internationalisation

### ISSUE THREAD

**Issue by mzeinstra** [creativecommons/chooser#553]
## Problem
I am getting calls from my local CC users that they cannot get chooser texts in their own language anymore

## Description
We need to add a language switch and add Transifex support

## Alternatives
Or revert to the old chooser

**Comment by shivpratikhande** [comment]
Yes, by adding the language switch support in the same way it is presented at this link (https://creativecommons.org/licenses/list.az#),

![image](https://github.com/user-attachments/assets/ecc23c9e-08b6-4c62-b874-1238b5e2f2bd)


 we can implement the same here as well 
 ![image](https://github.com/user-attachments/assets/8946076d-33ea-4ff1-ae37-ad0794ddf688)

**Comment by possumbilities** [comment]
Hi! I'm moving this to https://github.com/creativecommons/chooser/issues?q=state%3Aopen%20label%3A%22%F0%9F%9A%A7%20status%3A%20blocked%22 as there is currently a more stable refactor of the new Chooser underway. When that resolves we would then have room to start planning out how to incorporate Transifex into the Chooser and move things foward.

**Comment by mzeinstra** [comment]
Understood, however this does concern me. 
Do we have any numbers on how many times the translated versions of the chooser have been used in the past. 

Putting this on the backburner might send a wrong message to the community that helps translate CC's pages as well as the users that the licenses are not meant for them (in their languages).
